guided-match/guided-match-service.yaml

```diff
@@ -422,8 +422,10 @@ components:
         prePopulatedAnswerSource:
           type: string
           description: "Details of the source data used to pre-populate the answer - e.g. <system>|<json path>, so that the user can be told that the data came from user profile, org data and so on"
-        definedAnswers:
+        answerDefinitions:

```

Comment:
@davidhughes1-ccs - noticed this was inconsistent with the schema type, updated in both GM and DT APIs, hope that is ok with you?